code/src/utils/check-screen-size/useScreenSize.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+
+const useScreenSize = () => {
+  const [screenSize, setScreenSize] = useState({
+    width: window.innerWidth,
+    height: window.innerHeight
+  });
+
+  useEffect(() => {
+    const handleResize = () => {
+      setScreenSize({
+        width: window.innerWidth,
+        height: window.innerHeight
+      });
```

Comment:
There is following repeated code:

```tsx
{
        width: window.innerWidth,
        height: window.innerHeight
      }
```

I suggest creating following local helper function:

```tsx
const getWindowDimensions = () => {
    return {
         width: window.width,
         height: window.height
    }
}
```

Reply:
I have implemented said function, that returns the screen dimensions, in order to avoid code duplication.